independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/buildtool/maven/base/pom.tpl.qute.xml

```diff
@@ -206,6 +206,33 @@
                 <skipITs>false</skipITs>
             </properties>
         </profile>
+        <profile>
+            <id>native-with-agent</id>
+            <activation>
+                <property>
+                    <name>native-with-agent</name>
+                </property>
+            </activation>
+            <properties>
+                <quarkus.package.type>native</quarkus.package.type>
+                <skipITs>false</skipITs>
+            </properties>
+            <build>
+                <plugins>
```

Comment:
this is just for tests....but how about 'quarkus run' / `mvn quarkus:run` - should that be configured somehow?

Reply:
Yeah that should be done too. I'll look into it.

Reply:
@maxandersen I've looked into add quarkus run integration but I should be deferred to a later stage (we can create a follow up issue). Gradle `quarkusRun` does not seem to work at all. It starts Quarkus and then finishes the build, see [here](https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/Gradle.20quarkusRun.2C.20does.20it.20work.20at.20all.3F). Also, there are no integration tests under `gradle` that verify `quarkusRun` works at all. Native image agent integration with run targets should be added after that is fixed.